### PR TITLE
Select: update style if initially invisible

### PR DESCRIPTION
**Pull request by Leopoldthecoder**
Please makes sure these boxes are checked before submitting your PR, thank you!

* [ ] Make sure you follow Element's [Contributing Guide](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.md).
* [ ] Make sure you are merging your commits to `dev` branch.
* [ ] Add some descriptions and refer relative issues for you PR.

### CODE REVIEW

File: packages/select/src/select.vue [hunk 1]
```diff
@@ -363,7 +363,13 @@
       },
 
       getOption(value) {
-        const option = this.cachedOptions.filter(option => option.value === value)[0];
+        let option;
+        for (let i = 0, len = this.cachedOptions.length; i < len; i++) {
```
Comment (Leopoldthecoder) [inline packages/select/src/select.vue]: 需要找到最后一个符合条件的

Comment (QingWei-Li) [inline packages/select/src/select.vue]: 倒过来找第一个是不是就是最后一个了 🌚

Comment (Leopoldthecoder) [inline packages/select/src/select.vue]: 直接 `reverse` 的话会改变原数组，如果拷贝一份再 `reverse` 然后再找的话，性能可能还不如直接 for 循环

Comment (QingWei-Li) [inline packages/select/src/select.vue]: 不不 
`i = length; i--`  🌚

Comment (Leopoldthecoder) [inline packages/select/src/select.vue]: ……想自杀

**Pull request merged**